### ISSUE TITLE
Run without flags

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/n-heroku-tools#391]
Currently a bad network connection can halt development because the app won't bloody start without flags

How about, if NODE_ENV !== production, flags are cached in a .flags file and used as a last resort (or evena first resort if they're lazily updated) to get the app started

**Comment by matthew-andrews** [comment]
could we pull `.flags` in via n-makefile during `make install` and then fall back to that if flags not responding from the internet (plus a console warning)?

`nbt run` shouldn't write files because people who have to run apps over https need to `sudo make run` and any file written during run get really messed up permissions.

**Comment by wheresrhys** [comment]
Could do both, but only within nbt run if not sudo http://serverfault.com/questions/568627/can-a-program-tell-it-is-being-run-under-sudo

**Comment by matthew-andrews** [comment]
would prefer one way………  perhaps only do this if not sudo……?

**Comment by wheresrhys** [comment]
yeah. apps that need sudo probably have a few other network dependenices too anyway
